services/galley/src/Galley/API/Federation.hs

```diff
@@ -203,14 +203,27 @@ onConversationCreated domain rc = do
     pushConversationEvent Nothing event (qualifyAs loc [qUnqualified . Public.memId $ mem]) []
 
 onNewRemoteConversation ::
-  Member ConversationStore r =>
+  Members
+    '[ ConversationStore,
+       SubConversationStore
+     ]
+    r =>
   Domain ->
   F.NewRemoteConversation ->
   Sem r EmptyResponse
 onNewRemoteConversation domain nrc = do
   -- update group_id -> conv_id mapping
   for_ (preview (to F.nrcProtocol . _ProtocolMLS) nrc) $ \mls ->
-    E.setGroupIdForConversation (cnvmlsGroupId mls) (Qualified (F.nrcConvId nrc) domain)
+    case F.nrcSubConvId nrc of
+      Just subConvId ->
+        setGroupIdForSubConversation
```

Comment:
Probably out of scope but we should maybe be more consistent with imports. Pretty much every Effect interpreter everywhere is imported qualified as `E`, with a few exceptions here and there.

Reply:
Fixed it.